FinderCompass/src/main/java/atomicstryker/findercompass/common/DefaultConfigFilePrinter.java

```diff
@@ -162,22 +170,27 @@ else if (settingList.isEmpty())
                             }
                             else
                             {
-                                configInts[8] = -1;
+                                configInts[8] = 0;
                             }

```

Comment:
I have an idea to combine the special handling and the ore dictionary functionallity. -1 isn't the real problem, it is easy to change it to 0 automatically. But the problem is the free definition of damage.
My idea is to use a boolean flag (ignoreDamage) in CompassTargetData if no or -1 damage is set. In this case it is necessary to read the dictionary block names from damage 0 to upper with step width 1. If the ore dictionary name changes or there isn't a block, it will stop.
I think it isn't a good idea to search all blocks with different damage and different ore dictionary names for one identical needle.